sys/include/net/ng_fib.h

```diff
@@ -37,6 +37,14 @@ typedef struct rp_address_msg_t {
 #define FIB_MSG_RP_SIGNAL (0x99)     /**< message type for RP notifications */
 
 /**
+ * @brief entry used to collect available destinations
+ */
+typedef struct fib_destination_set_entry_t {
+    uint8_t dest[16];    /**< The destination address */
```

Comment:
sure, I would like to use `UNIVERSAL_ADDRESS_SIZE` for this magic number when #3022 is merged :) 
We can add a substitute now and switch it after #3022 has been merged.